### PR TITLE
Added color picker

### PR DESCRIPTION
**Pull request by namanhkapur**
- added subtitle to heading + COMP 413 tag (we can remove if we don't want it, I thought it looked good)
- added a color picker component

image

### CODE REVIEW

File: client/src/RoutingContainer.tsx [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-import Header from './header/containers/HeaderContainer';
-import Footer from './footer/components/Footer';
+import React, { FC } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import ScrollToTop from "./components/ScrollToTop";
+import CanvasPageContainer from "./canvas/containers/CanvasPageContainer";
+import AboutPage from "./about/components/About";
+import Header from "./header/containers/HeaderContainer";
+import Footer from "./footer/components/Footer";
+import ColorPicker from "./colorPicker/colorPicker";
```
Comment (almagee16) [inline client/src/RoutingContainer.tsx]: Looks like this line is the only new one.

File: client/src/header/components/Header.tsx [hunk 2]
```diff
@@ -32,29 +28,25 @@ const Header: FC<Props> = ({ onLogin, isLoggedIn, name, onLogout }) => {
...
       <Menu.Item>
-        <Link to='/about'>About</Link>
+        <Link to="/about">About</Link>
       </Menu.Item>
-      {isLoggedIn && (
-        <Menu.Item onClick={onLogout}>
-          Sign Out
-        </Menu.Item>
-      )}
+      {isLoggedIn && <Menu.Item onClick={onLogout}>Sign Out</Menu.Item>}
```
Comment (almagee16) [inline client/src/header/components/Header.tsx]: Why change this?

File: client/src/colorPicker/colorPicker.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import { SketchPicker } from "react-color";
+
+interface Props {}
+// http://casesandberg.github.io/react-color/
```
Comment (almagee16) [inline client/src/colorPicker/colorPicker.tsx]: Does this have to be here?

File: client/src/colorPicker/colorPicker.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import { SketchPicker } from "react-color";
+
+interface Props {}
```
Comment (almagee16) [inline client/src/colorPicker/colorPicker.tsx]: Can you try adding a callback here to see if the color is passed back upon a user's choice? Or do you plan to do that in another PR?

File: client/src/RoutingContainer.tsx (hunk 1 above)
Comment (rgreen312) [reply client/src/RoutingContainer.tsx]: Looks like you're converting all single quotes to double quotes... is there a reason for that?

File: client/src/colorPicker/colorPicker.tsx (hunk 3 above)
Comment (rgreen312) [reply client/src/colorPicker/colorPicker.tsx]: Let's leave it there for now for reference

File: client/src/RoutingContainer.tsx (hunk 1 above)
Comment (namanhkapur) [reply client/src/RoutingContainer.tsx]: I use the Prettier extension which sets a rule for frontend and it must enforce single quotes. Two ways to fix this: I can change the preference, or we all download Prettier so that we no longer have this problem as the style will be standardized

File: client/src/header/components/Header.tsx (hunk 2 above)
Comment (namanhkapur) [reply client/src/header/components/Header.tsx]: See comment above about Prettier

File: client/src/colorPicker/colorPicker.tsx (hunk 3 above)
Comment (namanhkapur) [reply client/src/colorPicker/colorPicker.tsx]: Agreed - it'll be helpful as a documentation source

File: client/src/RoutingContainer.tsx (hunk 1 above)
Comment (rgreen312) [reply client/src/RoutingContainer.tsx]: In general Prettier is great - could you just change that setting to use single quotes instead of double quotes? That's the typical convention

File: client/src/RoutingContainer.tsx [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
 
 const RoutingContainer: FC = () => (
   <Router>
     <ScrollToTop>
       <Header />
+      <ColorPicker />
```
Comment (rgreen312) [inline client/src/RoutingContainer.tsx]: Could we put this in `CanvasPageContainer`? Typically we only want to put pages in the routing container
